### DISCUSSION TITLE
Give feedback link in pull request conversations filter is unhelpful

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#12357] [Pull Requests] [answered]
The [Give feedback](https://support.github.com/contact/feedback?category=prs-and-code-review&subject=Conversations+menu) link in the conversations drop down takes me for a loop:

<img width="946" alt="image" src="https://user-images.githubusercontent.com/2119212/156629688-6e59efaa-49e3-4697-8027-13aa899c16d2.png">

https://support.github.com/contact/feedback?category=prs-and-code-review&subject=Conversations+menu
<img width="1010" alt="image" src="https://user-images.githubusercontent.com/2119212/156629839-c2eb9c8b-b47c-4e62-aa40-e89911bebc8e.png">

[Discussions](https://github.com/github/feedback/discussions)

Expected:
Just link me straight to https://github.com/github/feedback/discussions/new?category=pull-requests-feedback

**Comment by willsmythe** [comment, marked as answer]
Thanks for reporting! Because this feature has shipped/GAed, we are going to remove the link.
